testsuite/MDAnalysisTests/core/test_topologyattrs.py

```diff
@@ -121,6 +132,11 @@ class TestAtomids(TestAtomAttr):
     attrclass = tpattrs.Atomids
 
 
+# TODO: unusable because __init__ doesn't take any parameter
+# class TestAtomindices(TestAtomAttr):
```

Comment:
https://coveralls.io/builds/8502864/source?filename=miniconda%2Fenvs%2Fpyenv%2Flib%2Fpython2.7%2Fsite-packages%2FMDAnalysis%2Fcore%2Ftopologyattrs.py#L131

They're mostly covered through the tests [here](https://github.com/MDAnalysis/mdanalysis/blob/issue-363/testsuite/MDAnalysisTests/core/test_groups.py#L235).

If you wanted you could check that trying to change `indices` raises the right error, I get this:
```
----> 1 u.atoms.indices = 1
AttributeError: Atom indices are fixed; they cannot be reset
```

I'm not sure the `len` method is actually useful, so you could just delete it.